### DISCUSSION TITLE
Allow users to define which fields are in the loan item dialog

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5570] [Feature Requests]
Requested by Cal academy and several others

the current Loan Item Dialog (where you choose which and how many items go into a loan) only says the prep type and count available.  Users have requested to be able to add other fields from preparation into the dialogue to help narrow down which prep it is.
Several collections have a generic prep type, such as tissue, and another field that narrows down which type of tissue

**Comment by grantfitzsimmons** [comment]
From https://github.com/specify/specify7/issues/4028

> When selecting preparations for loan, we need to see more fields than just the preptype and count.
> 
> <img width="592" alt="Screenshot 2023-09-19 at 9 43 49 PM" src="https://github.com/specify/specify7/assets/37256050/c846b7db-bc1d-4f70-ba23-b27aefaadf57">

<img width="941" alt="image" src="https://github.com/specify/specify7/assets/37256050/0c231a01-9bce-4a2f-8138-26b3513bec0d">

> We'd like to add the text1 field from the preparation table as that describes the type of bone (or tissue or whatever).

**Requested By:** Jon Fong at California Academy of Sciences

**Requested By:** CSIRO

> Interesting as we were just talking about this yesterday. This is actually something we requested as well a very long time ago. That was pre-Asana though and I haven't been able to find a written request from our end. May have been in one of our very early on catch-up meetings where we discussed this.

![image](https://github.com/specify/specify7/assets/37256050/8f2524da-3dc6-4f15-86c0-e3132ed53d01)

**Comment by grantfitzsimmons** [comment]
**Requested by** Alexis at Geneva

> <img width="664" alt="image" src="https://github.com/user-attachments/assets/12597faf-d12d-4125-b296-4795f1134d7b" />
>
>I’d like to display another column which is missing for my user : preparation.description.

**Comment by philippeverley** [comment]
+1 for CAY and NOU herbarium

Custom columns would be much appreciated on the `Return Loan` form :+1:

**Comment by grantfitzsimmons** [comment]
Starting in 7.11.2, Specify will show the formatted Loan record based on the default table format for the table in the loan items dialog. See https://github.com/specify/specify7/pull/7341
